flexget/plugins/sites/wordpress.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+from future.moves.urllib.parse import urlencode
+
+import logging
+
+import re
+from flexget import plugin
+from flexget.event import event
+from flexget.plugin import PluginError
+from requests import Session
+from requests import Request, RequestException
+from requests.utils import dict_from_cookiejar, cookiejar_from_dict
+
+log = logging.getLogger('wordpress_auth')
+
+
+def _get_wp_login_data(username='', password='', redirect='/wp-admin/'):
```

Comment:
Is there any reason to have these as separate functions/classes is what I was thinking...

Reply:
no, there isn't
cleaning it up now